### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by michelleellen**
Update the bounties readme

### CODE REVIEW

**Comment by michelleellen** [comment]
@thehowl I think we should start with a fresh repo rather than the archive repo. I made it private in case you disagree with this approach. I have added the Readme file for your PR review. 

Thank you!

File: README.md [hunk 1]
```diff
@@ -1 +1,106 @@
-# gno_bounties
\ No newline at end of file
+# gno.land Bounty Program 
+
+We’ve provided a detailed guide on how to navigate the gno.land bounty ecosystem, and recommendations on how to participate and increase your chances of receiving the bounty.
+
+We have four different categories of bounties, and different bounty sizes linked to a monetary reward. Please take the time to read this information before starting to work on a bounty.
+
+### **Meta Bounties**
```
Comment (leohhhn) [inline README.md]:
```suggestion
### Meta Bounties
```

Adding a H3 already bolds the text

File: README.md [hunk 2]
```diff
@@ -1 +1,106 @@
...
+
+**Description:**
+
+Meta bounties focus on creating several impactful and reusable examples, dApps, and packages for Gno. Here is an initial example of this bounty [type](https://github.com/gnolang/gno/issues/3084)
```
Comment (leohhhn) [inline README.md]:
```suggestion
Meta bounties focus on creating several impactful and reusable examples, dApps, and packages for Gno. You can find an example of this bounty type [here](https://github.com/gnolang/gno/issues/3084).
```

File: README.md [hunk 3]
```diff
@@ -1 +1,106 @@
...
+
+These contributions should be:
+
+- Creative: Demonstrating unique and innovative use cases.
+- Straightforward: Easy to understand for developers of all skill levels.
+- Concise and Well-Documented: Clear source code with thorough documentation, serving as onboarding material and primary references for developers exploring Gno’s features and applications.
+- Specific: instead of tackling a wide array of functionality, the code should be focused on the objective and attempt to re-use as much as possible existing packages.
+
+Meta bounties range in value from $500 to $2,000, depending on the quality, complexity, and potential impact of the contribution. While the issue for the bounty is open, all submissions are entitled to the compensation with approval from the core team.
```
Comment (leohhhn) [inline README.md]:
```suggestion
Meta bounties range in value from $500 to $2,000, depending on the quality, complexity, and potential impact of the contribution. While the issue for the bounty is open, all submissions are entitled to the compensation (or part of the compensation) with approval from the core team.
```

META Bounties will generally be available for multiple submissions on different topics. I don't want people thinking they would get the full bounty amount if they add something small.

Basically, "META" as a concept is clear to us - it is a central place for multiple items (issues, submissions, etc), but not sure that that's the case externally, which is why I think we should explain this a bit more.

File: README.md [hunk 4]
```diff
@@ -1 +1,106 @@
...
+
+### **Scoped Bounties**
```
Comment (leohhhn) [inline README.md]:
```suggestion
### Scoped Bounties
```

File: README.md [hunk 5]
```diff
@@ -1 +1,106 @@
...
+
+**Description:**
+
+Scoped bounties are well-defined tasks with a detailed scope and clear expected outcomes. These bounties typically involve solving specific problems or implementing predefined features. An example of this bounty category is the [gno test -cover](https://github.com/gnolang/gno/issues/2908) bounties.
```
Comment (leohhhn) [inline README.md]:
```suggestion
Scoped bounties are well-defined tasks with a detailed scope and clear expected outcomes. These bounties typically involve solving specific problems or implementing predefined features. An example from this bounty category is the [`gno test -cover` bounty](https://github.com/gnolang/gno/issues/2908) bounties.
```

File: README.md [hunk 6]
```diff
@@ -1 +1,106 @@
...
+
+**Key Points:**
+
+- Each bounty has an exact monetary value, but can be shared between multiple participants. In the review process, a best effort will be made to distribute rewards fairly.
+- The deliverables and success criteria are clearly outlined in the bounty description.
+
+This category is straightforward and ideal for developers who prefer structured tasks with minimal ambiguity.
+
+### **Bounties for Competing Implementations**
```
Comment (leohhhn) [inline README.md]:
```suggestion
### Bounties for Competing Implementations
```

File: README.md [hunk 7]
```diff
@@ -1 +1,106 @@
...
+
+**Description:**
+
+This category aims to explore different approaches at solving a problem, in the form of multiple competing implementations. These are often core re-usable packages within gno.land, which can serve as a useful tool for the larger ecosystem. An example of this bounty category is the [UI Markdown templating](https://github.com/gnolang/gno/issues/2753) bounty.
```
Comment (leohhhn) [inline README.md]:
```suggestion
This category aims to explore different approaches at solving a problem, in the form of multiple competing implementations. These are often core re-usable packages within gno.land, which can serve as a useful tool for the larger ecosystem. An example from this bounty category is the [Markdown Library/Framework bounty](https://github.com/gnolang/gno/issues/2753).
```

File: README.md [hunk 8]
```diff
@@ -1 +1,106 @@
...
+**Key Points:**
+
+- The first implementations to be merged will receive an initial payout of 50% of the total bounty.
+- The remaining 50% is awarded based on adoption and usage over the course of the 6 months (or less) from the first implementations, incentivizing quality and user engagement.
+- Strong documentation and usability increase the chances of adoption and being recognized as the “winning” solution.
+- Implementations should have distinctive names.
+
+Monetary values for these bounties vary and are determined by the scope and potential impact of the feature.
+
+### **Bounties for Architectural Solutions**
```
Comment (leohhhn) [inline README.md]:
```suggestion
### Bounties for Architectural Solutions
```

File: README.md [hunk 9]
```diff
@@ -1 +1,106 @@
...
+**Key Points:**
+
+- Proposals are reviewed collectively, with the final solution potentially being a synthesis of multiple contributions.
+- Rewards are higher than other bounty categories, reflecting the complexity, collaboration, and level of effort required.
+- The core team will ultimately confirm which specification should be considered final, and give a go-ahead for the implementation.
+
+Due to the nature of these bounties, payouts are flexible and distributed based on contribution quality and input of each proposal in the final solution
+
+
+## **Bounty sizes**
```
Comment (leohhhn) [inline README.md]:
```suggestion
## Bounty sizes
```

File: README.md [hunk 10]
```diff
@@ -1 +1,106 @@
...
+| XS           | $500                  |
+| S            | $1000                 |
+| M            | $2000                 |
+| L            | $4000                 |
+| XL           | $8000                 |
+| XXL          | $16000                |
+| 3XL          | $32000                |
+
+
+## **Guide to Participating**
```
Comment (leohhhn) [inline README.md]:
```suggestion
##  A Guide to Participating
```

File: README.md [hunk 11]
```diff
@@ -1 +1,106 @@
...
+- Strong documentation and usability increase the chances of adoption and being recognized as the “winning” solution.
+- Implementations should have distinctive names.
+
+Monetary values for these bounties vary and are determined by the scope and potential impact of the feature.
+
+### **Bounties for Architectural Solutions**
+
+**Description:**
+
+These bounties focus on solving architectural challenges through creative proposals. The aim is to gather multiple innovative ideas and combine the best aspects into a single comprehensive specification - and after it’s approved, the implementation into code. An example of this type of bounty is the “[anti-squatting system](https://github.com/gnolang/gno/issues/2727)”.
```
Comment (leohhhn) [inline README.md]:
```suggestion
These bounties focus on solving architectural challenges through creative proposals. The aim is to gather multiple innovative ideas and combine the best aspects into a single comprehensive specification - and after it’s approved, the implementation into code. An example from this bounty category is the [anti-squatting system](https://github.com/gnolang/gno/issues/2727) bounty.
```

**Review by leohhhn — APPROVED** [review]
Looks good generally, just think we need to be a bit more clear for the META bounty section.

File: README.md [hunk 12]
```diff
@@ -1 +1,106 @@
...
+Meta bounties focus on creating several impactful and reusable examples, dApps, and packages for Gno. You can find an example of this bounty type [here](https://github.com/gnolang/gno/issues/3084).
+
+These contributions should be:
+
+- Creative: Demonstrating unique and innovative use cases.
+- Straightforward: Easy to understand for developers of all skill levels.
+- Concise and Well-Documented: Clear source code with thorough documentation, serving as onboarding material and primary references for developers exploring Gno’s features and applications.
+- Specific: instead of tackling a wide array of functionality, the code should be focused on the objective and attempt to re-use as much as possible existing packages.
+
+Meta bounties range in value from $500 to $2,000, depending on the quality, complexity, and potential impact of the contribution. While the issue for the bounty is open, all submissions are entitled to the compensation with approval from the core team.
```
Comment (thehowl) [inline README.md]:
```suggestion
Meta bounties range in value from $500 to $2,000, depending on the quality, complexity, and potential impact of the contribution. While the issue for the bounty is open, all submissions are entitled to compensation up to the bounty value, with approval from the core team.
```

**Pull request merged**